code-challenges.test.js

```diff
@@ -14,38 +14,129 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
+// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+
+describe("capitalize", () => {
+  it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized.", () => {
+    const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+  { name: "arthur dent", occupation: "a radio employee" }]
+    expect(capitalize(people)).toEqual(["Ford Prefect is a hitchhiker", "Zaphod Beeblebrox is president of the galaxy", "Arthur Dent is a radio employee"])
+  })
+})
+
+
+//======================== GOOD ERROR FAILURE ===============================//
+// ● sumCubed › takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
+
+// ReferenceError: sumCubed is not defined
+//===========================================================================//
 
 
 // b) Create the function that makes the test pass.
+//========================== PSEUDO CODE ====================================//
+//create a function named 'capitalize'
+//must take in an array of object as a parameter, we'll call it 'arr'
+//declare a new variable called 'newArr' and set it to arr.map to iterate through the values of the array
+//declare a new variable called 'name' and set it to the values.name.split.map which would refer to the 'name' key of the object and will split the name by spaces into 2 seperate indexes
+//iterate through the array of the 'name' key and refer to the first letter of each index by using .charAt() method and refer to its 0 index
+//capitalize it by using .toUpperCase() method, and add the value back into the array by using .substring(1) which would add the rest of the word without the first letter included
+//return the variable 'name' which contains the capitalized letters and use .join() method to join the first name and last name into 1 index
+//return the name plus the word "is" and refer to the key value of occupation to print out that value, which would be the whole completed sentence. 
+//===========================================================================//
+
+
+const capitalize = (arr) => {
+    let newArr = arr.map(v => {
+      let name = v.name.split(" ").map(val => val.charAt(0).toUpperCase() + val.substring(1) 
+      )
+      return v = name.join(" ") + ' is ' +  v.occupation 
+    })
+    return newArr
+}
+
+
+//TEST PASSED! 
+
```

Comment:
Great work flow. For your future studies please consider where you can combine steps